packages/rn-tester/js/examples/SectionList/SectionListExample.js

```diff
@@ -288,9 +332,14 @@ const styles = StyleSheet.create({
     alignSelf: 'center',
     fontSize: 7,
   },
+  emptyList: {
+    textAlign: 'center',
+    marginVertical: 20,
+    fontSize: 14,
+  },
 });
 
-exports.title = 'SectionList';
+exports.title = '<SectionList>';
```

Comment:
We are removing the angle brackets from title.

```suggestion
exports.title = 'SectionList';
```